schemas/maas-backend/customers/favorite-locations/add/response.json

```diff
@@ -0,0 +1,56 @@
+{
+  "$id": "http://maasglobal.com/maas-backend/customers/favorite-locations/add/response.json",
+  "description": "MaaS customer favorite location adding response",
+  "type": "object",
+  "properties": {
+    "favoriteLocation": {
```

Comment:
We could factor out `favoriteLocation` into a component, and then reference the component in these request/response schemas.